ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-commands.sh

```diff
@@ -66,3 +66,7 @@ compute:
       zones: ${ZONES_STR}
   replicas: ${workers}
 EOF
+
+if [[ -s "${SHARED_DIR}/customer_vpc_subnets.yaml" ]]; then
```

Comment:
Suggest to introduce a dedicated step - `ipi-conf-ibmcloud-sharednetwork` to modify install-config.yaml to avoid affect the existing steps.